src/mame/audio/8080bw.h

```diff
@@ -0,0 +1,44 @@
+// license:BSD-3-Clause
+/***************************************************************************
+
+    8080-based black and white sound hardware
+
+****************************************************************************/
+#ifndef MAME_AUDIO_8080BW_H
+#define MAME_AUDIO_8080BW_H
+
+#pragma once
+
+#include "machine/timer.h"
+#include "sound/discrete.h"
+#include "sound/samples.h"
+#include "sound/sn76477.h"
+
+class cane_audio_device : public device_t
+{
+public:
+	cane_audio_device(machine_config const &mconfig, char const *tag, device_t *owner, u32 clock = 0);
+
+	void cane_sh_port_1_w(u8 data);
+	void cane_music_w(u8 data);
+	void cane_76477_en_w(u8 data);
+	void cane_76477_dis_w(u8 data);
```

Comment:
You don't need the `cane_` prefix on all of these - it's implied in the device name.

Reply:
I just noticed that `cane_76477_en_w` and `cane_76477_dis_w` aren't called from anywhere - is that intentional?